### PR TITLE
Postgres configuration

### PR DESCRIPTION
**Pull request by katarzynajanus**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -22,12 +22,14 @@ dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
 	implementation 'org.springframework.boot:spring-boot-starter-security'
 	implementation 'org.springframework.boot:spring-boot-starter-web'
+	implementation group: 'org.junit', name: 'junit5-engine', version: '5.0.0-ALPHA'
 	compileOnly 'org.projectlombok:lombok'
```
Comment (MJanus2) [inline build.gradle:26]: Please keep the same convention as it is in `'org.springframework.boot:spring-boot-starter-web'` in this case it should be `'org.junit:junit5-engine'`

File: build.gradle [hunk 2]
```diff
@@ -22,12 +22,14 @@ dependencies {
...
 	runtimeOnly 'com.h2database:h2'
 	runtimeOnly 'org.postgresql:postgresql'
 	annotationProcessor 'org.projectlombok:lombok'
 	testImplementation 'org.springframework.boot:spring-boot-starter-test'
 	testImplementation 'org.springframework.security:spring-security-test'
+	testImplementation group: 'org.mockito', name: 'mockito-core', version: '2.1.0'
 }
```
Comment (MJanus2) [inline build.gradle:33]: Please create ext {
mockitoVersion = '2.1.0'
} 
and then implement it here as org.mockito:mockito-core:$mockitoVersion
same for JUnit5

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -1 +1,6 @@
+spring.datasource.url=jdbc:postgresql://localhost:5432/realestate
```
Comment (MJanus2) [inline src/main/resources/application.properties:1]: Please convert it to yaml file. 
How to do it I found comment in the GitHub:
_for intellij-IDEA : go to file > settings > plugins then search for a plugin called: Properties to YAML Converter. Install it and After that restart your IDE then right click the application.properties file and you will get an option called "convert to yaml"_

Why:
https://www.baeldung.com/spring-boot-yaml-vs-properties

**Review by MJanus2 — CHANGES_REQUESTED** [review]
Please squash all commits and add with one meaningfull commit. 
Please remove .gradle/.idea and build folder from the PR. It can be created during gradle build so it is no needed to be in the repository.

File: build.gradle (hunk 1 above)
Comment (katarzynajanus) [reply build.gradle:26]: updated as testImplementation 'org.mockito:mockito-core:5.3.1' and 	implementation 'org.junit:junit5-engine:5.0.0-ALPHA' as I got compile error

**Review by MJanus2 — APPROVED** [review]

**Pull request merged**